deploy/charts/rook-ceph/templates/securityContextConstraints.yaml

```
@@ -0,0 +1,91 @@
+# scc for the Rook and Ceph daemons
+# for creating cluster in openshift
+{{- if .Values.openshift }}
```

instead of a new variable, you can do as below for auto discovery

```suggestion
{{- if .Capabilities.APIVersions.Has "security.openshift.io/v1/SecurityContextConstraints" }}
apiVersion: security.openshift.io/v1
kind: SecurityContextConstraints
...
{{- end }}
```

https://docs.helm.sh/docs/chart_template_guide/builtin_objects/

---

That great